src/Symfony/Bridge/Doctrine/Test/DoctrineTestHelper.php

```diff
@@ -14,6 +14,7 @@
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
 use Doctrine\ORM\EntityManager;
+use PHPUnit\Framework\TestCase;
```

Comment:
@stof Actually we want 5.4.3 as this is the first one with the correct namespace (https://github.com/sebastianbergmann/phpunit/blob/5.4.3/src/ForwardCompatibility/TestCase.php#L11). 5.4.2 had the lowercase one. https://github.com/sebastianbergmann/phpunit/blob/5.4.3/src/ForwardCompatibility/TestCase.php#L11